server/graphql/v1/mutations.js

```diff
@@ -451,6 +451,37 @@ const mutations = {
       }
     },
   },
+  sendEmailToAdmins: {
```

Comment:
@Betree This is the mutation referenced in the frontend for the backend, the only difference is that `sendEmailToAdmins` sends an email, yet `editCoreContributors` doesn't.